contracts/terraswap_factory/README.md

```diff
@@ -18,6 +19,7 @@ The factory contract can perform creation of terraswap pair contract and also be
 ## ExecuteMsg
 
 ### `update_config`
+The factory contract owner can change relevant code IDs for future pair contract creation.
```

Comment:
It can also change `owner` and `token_code_id`(lp token)

Reply:
[6cc17a5](https://github.com/terraswap/terraswap/pull/68/commits/6cc17a52c045b5961c2389154bdfe9045d649bbe)